lib/matplotlib/tests/test_arrow_patches.py

```diff
@@ -12,7 +12,7 @@ def draw_arrow(ax, t, r):
 
 
 @image_comparison(['fancyarrow_test_image'],
-                  tol=0.012 if platform.machine() == 'arm64' else 0)
+                  tol=0.012 if platform.machine() != 'x86_64' else 0)
```

Comment:
Optional: I think this is easier to read the other way round 

```suggestion
                  tol=0 if platform.machine() == 'x86_64' else 0.012)
```